importer/RoRo.py

```diff
@@ -1,29 +1,117 @@
 
     def set_adm_location(self):
-        return
+        counties = self.data_files["counties"]
+        adm_code = self.judetul_iso
+        county_item = utils.get_item_from_dict_by_key(dict_name=counties,
+                                                      search_term=adm_code,
+                                                      search_in="iso_code")
+        if len(county_item) == 1:
+            self.add_statement("located_adm", county_item[0])
+        else:
+            self.add_to_report("judetul_iso",
+                               self.judetul_iso,
+                               "located_adm")
 
-    def set_no(self):
-        self.add_statement("romanian_monument_id", self.cod)
+    def set_location(self):
+        """
+        Set Location property from art wlinked in localitate.
 
-    def __init__(self, db_row_dict, mapping, data_files, existing):
-        Monument.__init__(self, db_row_dict, mapping, data_files, existing)
-        self.update_labels()
-        # self.exists("sq")
+        Run this after set_adm_location. localitate can
+        contain several links (we take the 1st which seems to
+        be the most granular one) and a mix of administrative
+        types. Compare with admin location so that they're not
+        the same.
+        """
+        if self.has_non_empty_attribute("localitate"):
+            loc_item = None
+            if utils.count_wikilinks(self.localitate) > 0:
+                loc_link = utils.get_wikilinks(self.localitate)[0]
+                loc_item = utils.q_from_wikipedia("ro", loc_link.title)
+                adm_item = self.get_statement_values("located_adm")
+                if loc_item and loc_item != adm_item[0]:
+                    self.add_statement("location", loc_item)
+
+            if not loc_item:
+                self.add_to_report("localitate", self.localitate, "location")
+
+    def set_wlm_id(self):
+        """
+        Set the WLM ID.
+
+        Do _not_ use the Romania LMI code (P1770)
+        as it's made up and has no relevance to the
+        real world.
+        """
+        ccode = self.mapping["country_code"].upper()
+        wlm_id = "{}-{}".format(ccode, self.cod)
+        self.add_statement("wlm_id", wlm_id)
+
+    def update_descriptions(self):
+        adm_code = self.judetul_iso
+        counties = self.data_files["counties"]
+        county_item = utils.get_item_from_dict_by_key(dict_name=counties,
+                                                      search_term=adm_code,
+                                                      return_content_of="itemLabel",
+                                                      search_in="iso_code")
+        if len(county_item) == 1:
+            place_name = "{}, Romania".format(county_item[0])
+        else:
+            place_name = "Romania"
+        desc = "heritage site in {}".format(place_name)
+        self.add_description("en", desc)
+        self.add_disambiguator(utils.remove_markup(str(self.cod)))
```

Comment:
what is the reason for the `utils.remove_markup()` here when you didn't use it in `set_monuments_all_id`?

Reply:
No reason. I think it's a leftover from when there was something else inside of this. Removed it.